kroxylicious-api/src/main/antlr4/io/kroxylicious/proxy/metadata/selector/Selector.g4

```diff
@@ -0,0 +1,87 @@
+grammar Selector;
```

Comment:
is this your own implementation or is this based on something (grammar used by Kubernetes?)